src/intervals/arithmetic.jl

```diff
@@ -120,15 +99,9 @@ function *(x::S, a::Interval{T}) where {T, S<:AbstractFloat}
     (iszero(a) || iszero(x)) && return zero(Interval{T})
 
     if x ≥ 0.0
-        c = @round(a.lo*x, a.hi*x)
-    else
-        c = @round(a.hi*x, a.lo*x)
-    end
-    
-    if(typeof(c) != Interval{T})
-        convert(Interval{T}, c)
+        c = @round(Interval{T}, a.lo*x, a.hi*x)
     else
-        return c
+        c = @round(Interval{T}, a.hi*x, a.lo*x)
```

Comment:
`c` is never used so a `return` statement would be slightly clearer.

Also I don't know if we have official guideline for the package, but I think explicit `return` statement are generally preferred.